lib/Ohmbrewer_Screen.cpp

```diff
@@ -398,22 +382,21 @@ unsigned long Ohmbrewer::Screen::captureButtonPress() {
     char printx [10];
     char printy [10];
     char status [40];
-    
+
     // a point object holds x y and z coordinates
     TSPoint p =_ts->getPoint();
     //According to Particle forums, the read below is necessary to get touch to work on Photon
     _ts->readTouchY();
 
-  
     // we have some minimum pressure we consider 'valid'
     // pressure of 0 means no pressing!
-    if (p.z > MINPRESSURE || p.z < MAXPRESSURE) {
+    if (p.z < MINPRESSURE || p.z > MAXPRESSURE) {
```

Comment:
If this is the same as #75, maybe we can get that confirmed and merged in? :wink: